libs/wingc/src/jsify.rs

```diff
@@ -1041,6 +963,15 @@ impl<'a> JSifier<'a> {
 		for (method_name, method_refs) in refs {
 			bind_method.open(format!("if (ops.includes(\"{method_name}\")) {{"));
 			for (field, ops) in method_refs {
+				// field might be a dotted accessor, like foo.bar.baz, in which case
```

Comment:
Can you explain this fix? Is it related to this PR or something else? Can you give an example?

Reply:
Let me remove this hack and create a separate issue - I have a feeling this isn't the right fix, but it might require more investigation.

Reply:
But TL;DR some bindings aren't generated correctly when accessing members of a resource (but the only case we have of it happening is with `api.url`)